pages/instructorCourseAdminSharing/instructorCourseAdminSharing.html.js

```diff
@@ -0,0 +1,257 @@
+                  sanitize: false
+                })
+            });
+        </script>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", resLocals)}
+        <div id="content" class="container-fluid">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white d-flex">
+                Course Sharing Info
+            </div>
+            <table class="table table-sm table-hover two-column-description">
+              <tbody>
+                <tr><th>Sharing Name</th><td>${
+                  sharing_name === null ? chooseSharingNameButton(resLocals) : sharing_name
+                }</td></tr>
+                <tr><th>Sharing ID</th>
+                  <td>
+                  ${sharing_id}
+                  <button class="btn btn-xs btn-secondary mx-2" onclick="navigator.clipboard.writeText('${sharing_id}');">
+                    <i class="fa fa-copy"></i>
+                    <span>Copy</span>
+                  </button>
+                  <form name="sharing-id-regenerate" method="POST" class="d-inline">
+                    <input type="hidden" name="__action" value="sharing_id_regenerate">
+                    <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}">
+                    <button role="button" type="submit" class="btn btn-xs btn-secondary">
+                      <i class="fa fa-rotate"></i>
+                      <span>Regenerate</span>
+                    </button>
+                  </form>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+
+          <div class="card mb-4">
+          <div class="card-header bg-primary">
+            <div class="row align-items-center justify-content-between">
+              <div class="col-auto">
+                <span class="text-white">Sharing Sets</span>
+              </div>
+              <div class="col-auto">
+                <button type="button" class="btn btn-light btn-sm ml-auto" id="courseSharingSetAdd" tabindex="0"
+                  data-toggle="popover" data-container="body" data-html="true" data-placement="auto" title="Create Sharing Set"
+                  data-content="${addSharingSetPopover(resLocals)}"
+                  data-trigger="manual" onclick="$(this).popover('show')"
+                >
+                  <i class="fas fa-plus" aria-hidden="true"></i>
+                  <span class="d-none d-sm-inline">Create Sharing Set</span>
+                </button>
+              </div>
+            </div>
+          </div>
+          <table class="table table-sm table-hover table-striped">
+            <thead>
+              <th>Sharing Set Name</th>
+              <th>Shared With</th>
+            </thead>
+            <tbody>
+              ${sharing_sets.map(
+                (sharing_set) => html`
+                  <tr>
+                    <td class="align-middle">${sharing_set.name}</td>
```

Comment:
This could probably be saved for a future PR, but it would be excellent to be able to click a sharing set name and go to a page that shows all questions in a set, and maybe some details about the courses they're shared with?

Reply:
Ya that sounds awesome! Let me make an issue for this and for other features that need to follow up this one in separate PRs, I can start organizing them all into a project.